mmv1/products/cloudbuildv2/Repository.yaml

```diff
@@ -0,0 +1,105 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: Repository
+base_url: projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/repositories
+create_url: projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/repositories?repositoryId={{name}}
+self_link: projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/repositories/{{name}}
+immutable: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/build/docs'
+  api: 'https://cloud.google.com/build/docs/api/reference/rest'
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+description: "A repository associated to a parent connection."
+exclude_tgc: true
+examples:
+# These tests depend on secrets stored in a separate project, so we prefer not
+# to show them in the docs.
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudbuildv2_repository_ghe'
+    primary_resource_id: 'primary'
+    skip_docs: true
+    vars:
+      connection_name: 'connection'
+      repository_name: 'repository'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudbuildv2_repository_github'
+    primary_resource_id: 'primary'
+    skip_docs: true
+    vars:
+      connection_name: 'connection'
+      repository_name: 'repository'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudbuildv2_repository_gle'
+    primary_resource_id: 'primary'
+    skip_docs: true
+    vars:
+      connection_name: 'connection'
+      repository_name: 'repository'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudbuildv2_repository_ghe_doc'
+    skip_test: true
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudbuildv2_repository_github_doc'
+    skip_test: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  encoder: templates/terraform/encoders/cloudbuildv2_repository.go.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: location
+    description: The location for the resource
+    url_param_only: true
+    immutable: true
+    default_from_api: true  # EXTRACT_FROM_FIELD_IF_EMPTY
+  - !ruby/object:Api::Type::ResourceRef
+    name: parent_connection
+    resource: connection
+    imports: name
+    description: The connection for the resource
+    url_param_only: true
+    required: true
+    immutable: true
+    diff_suppress_func: 'tpgresource.CompareResourceNames'
```

Comment:
https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9631-old..auto-pr-9631#diff-0384c5e5088c66b29f694cd9ae573be43e310c2a13688d35faddd184864c7b9aR64

Is this diff intentional?